src/platform/silabs/platformAbstraction/WiseMcuSpam.cpp

```diff
@@ -92,8 +92,6 @@ SilabsPlatform::SilabsButtonCb SilabsPlatform::mButtonCallback = nullptr;
 
 CHIP_ERROR SilabsPlatform::Init(void)
 {
-    mButtonCallback = nullptr;
```

Comment:
Why this deletion?

Reply:
Right above we have:
`SilabsPlatform::SilabsButtonCb SilabsPlatform::mButtonCallback = nullptr;`